asreview/io/ris_writer.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019-2020 The ASReview Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rispy
+import pandas as pd
+import copy as cp
+
+
+def write_ris(df, fp):
+    """RIS file writer
+    Parameters
+    ----------
+    df: pandas.Dataframe
+        Dataframe to convert and export.
+    fp: str, pathlib.Path
+        File path to the RIS file, if exists.
+    """
+    # # Choose only "included" records
+    # df = df[df.included.astype('str').str.contains('1')]
+
+    # print("Writer DF is:\n",df[["authors","keywords","included"]])
+    # Turn pandas DataFrame into records (list of dictionaries) for rispy
+    records = df.to_dict('records')
+    # print("records are:\n", records)
+
+    ########################################################
+    # For each record, update and verify RIS list type tags:
+    # AU - authors
```

Comment:
A1,A2,A3 are automatically included in this list.

Reply:
I removed the comment completely to make it less cluttered